### DISCUSSION TITLE
wallet security key issue

### DISCUSSION THREAD

**Discussion by SerkaanD** [IBAX-io/go-ibax#1874] [Bug Submission]
When we create a wallet on a different computer after opening the wallet, the security key does not work. Also, the backup account private key created in the wallet seemed easy to steal. I don't think it's safe enough.

**Comment by yolandadadada** [comment]
Why do you feel insecure?

**Comment by SerkaanD** [comment]
318 / 5.000
Çeviri sonuçları
Hello;
I think there is a high probability that the imported codes will be stolen
in creating a backup account. It seems difficult to protect the downloaded
text file in case the computers are hacked. I think that if anyone gets
their hands on it, they can open a backup account and transfer everything
in the wallet to their own account.

yolandadada ***@***.***>, 11 Şub 2022 Cum, 10:50 tarihinde
şunu yazdı:
